src/backend/parser/parse_utilcmd.c

```diff
@@ -396,13 +396,19 @@ transformCreateStmt(CreateStmt *stmt, const char *queryString)
 													 likeDistributedBy, bQuiet);
 	}
 
+	/*
+	 * CBDB: for a foreign table, do not inherit source table's distribution policy.
+	 * It should be decided by OPTIONS, ex: mpp_execute all segments.
+	 */
+#if 0
 	if (IsA(stmt, CreateForeignTableStmt))
 	{
 		DistributedBy *ft_distributedBy = ((CreateForeignTableStmt *)stmt)->distributedBy;
 		if (ft_distributedBy || likeDistributedBy)
 			stmt->distributedBy = transformDistributedBy(pstate, &cxt, ft_distributedBy,
 														 likeDistributedBy, bQuiet);
 	}
```

Comment:
What's the behavior if the command specifies the distribution policy? Shall we throw an error or warning to the user, other than swallowing its definition.